modules/idlgen/core/src/main/scala/avro/AvroSrcGenerator.scala

```diff
@@ -105,11 +109,10 @@ object AvroSrcGenerator extends SrcGenerator {
 
     val packageLines = Seq(schemaLines.head, "")
 
-    val importLines = Seq(
-      "import freestyle.rpc.internal.encoders.avro.bigdecimal._",
-      "import freestyle.rpc.internal.encoders.avro.javatime._",
-      "import freestyle.rpc.protocol._"
-    )
+    val importLines =
+      ("import freestyle.rpc.protocol._" :: marshallersImports
+        .map(_.marshallersImport)
+        .map("import " + _)).sorted
```

Comment:
Should we check if the user has specified the `import` reserved word before adding it?